packages/contracts/contracts/interfaces/ILenderManager.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity >=0.8.0 <0.9.0;
+
+// SPDX-License-Identifier: MIT
+
+interface ILenderManager {
```

Comment:
what if instead we made this an abstract contract that way we could make it extend the `IERC721` interface and not have to have `TellerV2` cast it when checking the `.ownerOf`
```suggestion
abstract contract ILenderManager is IERC721Upgradeable {
```